hibernate-rx-core/src/test/java/org/hibernate/rx/ReactiveSessionTest.java

```diff
@@ -102,12 +101,14 @@ public void init() {
 		session = sessionFactory.unwrap( RxHibernateSessionFactory.class ).openRxSession();
 	}
 
+	@After
+	public void tearDown() {
+		dropTable().whenComplete( (r, e) -> { sessionFactory.close(); } );
```

Comment:
If this is async you probably need to tell the test context about it.

Reply:
The thing is that this is not a test. It doesn't seem to work if I use TestContext like in the other tests. Is there a proper way to do it in the `@After` methods?

Reply:
@vietj says it should work, use the `TestContext` like in the other test methods.

Reply:
it should work given you provide a `TestContext` argument in the method

Reply:
This is a vertx-junit test, so it _is_ supported:

```java
@After
    public void after(TestContext context) {
      vertx.close(context.asyncAssertSuccess());
    }
```

Reply:
Thanks, I can see that it works now. Probably, there was something else wrong when I tried before.

Reply:
I've added a fix. Thanks